### PR TITLE
[zh-cn]: create the translation of `tabs.sendRequest()` method

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/sendrequest/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: tabs.sendRequest()
+slug: Mozilla/Add-ons/WebExtensions/API/tabs/sendRequest
+l10n:
+  sourceCommit: 43e3ff826b7b755b05986c99ada75635c01c187c
+---
+
+{{AddonSidebar}}
+
+**警告：** 此方法已被弃用。请使用 {{WebExtAPIRef("tabs.sendMessage()")}} 替代。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/sendrequest/index.md:10]:
```suggestion
> **警告：** 此方法已被弃用。请使用 {{WebExtAPIRef("tabs.sendMessage()")}} 替代。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/sendrequest/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+### 参数
+
+- `tabId`
+  - : `integer`。
+- `request`
+  - : `any`。
+
+### 返回值
+
+一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，内容脚本中处理消息的处理程序发送 JSON 响应对象时其会兑现，如果内容脚本未发送响应，则返回空参数。如果连接到指定标签页时出现错误或其他任何错误，promise 将会被拒绝并返回错误信息。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/sendrequest/index.md:34]:
```suggestion
一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，内容脚本中处理消息的处理器发送 JSON 响应对象时其会兑现，如果内容脚本未发送响应，则不兑现任何内容。如果连接到指定标签页时出现错误或其他任何错误，promise 将以错误信息拒绝。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/sendrequest/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+### 参数
+
+- `tabId`
+  - : `integer`。
+- `request`
+  - : `any`。
+
+### 返回值
+
+一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，内容脚本中处理消息的处理器发送 JSON 响应对象时其会兑现，如果内容脚本未发送响应，则不兑现任何内容。如果连接到指定标签页时出现错误或其他任何错误，promise 将以错误信息拒绝。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/sendrequest/index.md:34]:
```suggestion
一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，内容脚本中的消息句柄发送响应对象时其会兑现，如果内容脚本未发送响应，则不兑现任何内容。如果连接到指定标签页时出现错误或其他任何错误，promise 将以错误信息拒绝。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/sendrequest/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+### 参数
+
+- `tabId`
+  - : `integer`。
+- `request`
+  - : `any`。
+
+### 返回值
+
+一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，内容脚本中的消息句柄发送响应对象时其会兑现，如果内容脚本未发送响应，则不兑现任何内容。如果连接到指定标签页时出现错误或其他任何错误，promise 将以错误信息拒绝。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/sendrequest/index.md:34]: 就翻译为“处理器”：https://github.com/mdn/translated-content/blob/main/docs/zh-cn/glossary.md#javascript

```suggestion
一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，内容脚本中消息的处理器发送响应对象时其会兑现，如果内容脚本未发送响应，则不兑现任何内容。如果连接到指定标签页时出现错误或其他任何错误，promise 将以错误信息拒绝。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**